test/lit/passes/optimize-instructions-exact.wast

```diff
@@ -0,0 +1,66 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+
+;; RUN: wasm-opt %s -all --disable-custom-descriptors --optimize-instructions -S -o - | filecheck %s
+
+(module
+ ;; CHECK:      (type $super (sub (struct)))
+ (type $super (sub (struct)))
```

Comment:
As there is no sub, perhaps this could be called `struct`?

Reply:
I'll use `$foo` since it doesn't matter that it's a struct in particular, either.